R/process_data.R

```diff
@@ -108,8 +114,6 @@ process_excluded_companies <- function(data, company_exclusion, technologies) {
 process_df_price <- function(data, technologies, sectors, start_year, end_year) {
   data_processed <- data %>%
     check_technology_availability(expected_technologies = technologies) %>%
-    dplyr::filter(year >= start_year) %>%
```

Comment:
is this handled by using an inner join later on @MirijaHa ?

Reply:
The filtering by start year (and also end year) is also done in l 115, so I deleted this line 111.
Line 112 looks useless to me. It basically checks that data$year > start_year. Since we ensure that by filtering anyway I removed the line. Then again there probably was a reason why this function was even written, so maybe I am overlooking something? 🤔

Reply:
no seems fine then